internal/nonodo/nonodo.go

```diff
@@ -153,7 +154,17 @@ func NewSupervisorHLGraphQL(opts NonodoOpts) supervisor.SupervisorWorker {
 		db = sqlx.MustConnect("postgres", connectionString)
 	} else {
 		slog.Info("Using SQLite ...")
-		db = sqlx.MustConnect("sqlite3", opts.SqliteFile)
+		sqliteFile := opts.SqliteFile
```

Comment:
I think you can move this code to a function that returns db and reuse it below, what do you think ?

Reply:
For sure, I'll change it.